docs/common_commcare_mobile_dev_workspace_build_errors.rst

```diff
@@ -0,0 +1,202 @@
+Common CommCare Mobile Dev Workspace Build Errors
```

Comment:
It's not clear to my that mobile docs should be in the HQ repo. I think the benefit is that we only need 1 docs site and infratructure around that? The downside is it violates the "keep docs close to code" principle - since this will often be the only reason mobile devs come to edit the HQ repo.